src/Lumi/Components/DropdownButton.purs

```diff
@@ -269,6 +272,14 @@ dropdownIcon props =
 
 foreign import checkIsEventTargetInTree :: EffectFn2 Node Event Boolean
 
+getDimensions :: HTML.HTMLElement -> Effect { width :: Number, height :: Number }
+getDimensions el = do
+  { width, height } <- HTML.getBoundingClientRect el
+  pure
+    { width: width
+    , height: height
+    }
```

Comment:
Not a big deal, but we could also just include height and width in the `getAbsolutePosition` function's response, and/or include `top` and calculate height/width. But this works fine too.

Reply:
I just wanted a function that was explicitly clear it was for getting dimensions... but can go either way on this too.

Reply:
I agree, I quite like the separation here, especially since `getAbsolutePosition` has a single clear responsibility. I might consider inlining this function, i.e. just using `getBoundingClientRect` directly and pattern matching to get just the fields you're interested in and discard the rest, but again it's not a big deal.